release_notes/40-release-notes.adoc

```diff
@@ -472,6 +473,17 @@ Release date: 31 May 2023
 //ROX-17189
 * Fixed an issue that caused frequent Central pod restarts with the `context deadline exceeded` error.
 
+[id="resolved-in-version-403"]
+=== Resolved in version 4.0.3
+
+Release date: 13 July 2023
+
+* Fixed an issue where {product-title-short} sometimes incorrectly attempted to install PodSecurityPolicies (PSPs), causing the installation to fail because PSPs are not supported in Kubernetes version 1.25 and later. (ROX-17796, ROX-16652)
+* Fixed an issue with upgrades failing because {product-title-short} applied PSPs, even when not enabled. (ROX-17771, ROX-17734)
```

Comment:
Wording suggestion, including the use of the preferred term, "[update](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update)":

```suggestion
* Fixed an issue where updates failed because {product-title-short} applied PSPs, even when not enabled. (ROX-17771, ROX-17734)
```

Reply:
ACS is a little weird because the existing docs use "upgrade," so to use "update" in this context would be inconsistent...but it's a good point that we should eventually change our language to match the standards.